tests/contrib/logging/test_tracer_logging.py

```diff
@@ -91,7 +91,8 @@ def test_unrelated_logger_loaded_first(
 assert custom_logger.parent.name == 'root'
 assert custom_logger.level == logging.WARN
 
-ddtrace_logger = logging.getLogger('ddtrace')
+from ddtrace._logger import logging as ddtrace_logging
```

Comment:
If we are not unloading the `logging` module, then `ddtrace_logging` should be exactly `logging`